### PR TITLE
KAA-1123: Upgrading your instance, 0.9.x to 0.10.x - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md [hunk 1]
```diff
@@ -1,48 +1,39 @@
...
-```bash
- sudo ./upgradeKaa081to090.sh
-```
-    
- >**Note:** 
- >
- > During installation of debian package the script may ask you about resolving conflicts in the /usr/lib/kaa-node/conf/ folder,
- > be attentive about merging configurations because incorrect server setting can lead the Kaa server to malfunction.
\ No newline at end of file
+		sudo ./upgradeKaa081to090.sh
```
Comment (sergeyevd) [inline doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: Filename in command does not match filename from step 2.

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md [hunk 2]
```diff
@@ -1,48 +1,39 @@
...
+>In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise, unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing, shall any Contributor be liable to You for damages, including any direct, indirect, special, incidental, or consequential damages of any character arising as a result of this License or out of the use or inability to use the Work (including but not limited to damages for loss of goodwill, work stoppage, computer failure or malfunction, or any and all other commercial damages or losses), even if such Contributor has been advised of the possibility of such damages.”
+{:.caution}
+
+To update your Kaa server from version 0.9.x to 0.10.x:
+
+1. [Download](http://www.kaaproject.org/download-kaa/) the Kaa 0.10.0 Debian package.
+
+2. Download the [upgradeKaa090to0100.sh](https://github.com/kaaproject/kaa/blob/v0.10.0/server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh) and [data-migration.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/db-migration.one-jar.jar) to upgrade the databases.
```
Comment (vadimol) [inline doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: The links are not valid.
1) https://github.com/kaaproject/kaa/blob/v0.10.0/server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh

2) http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/db-migration.one-jar.jar

**Review by vadimol — CHANGES_REQUESTED** [review]

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: Fixed.

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: The links will be valid after the build.

Comment (hunch7) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: At least this is what @Kirill380 told me :)

**Comment by Kirill380** [comment]
LGTM

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md [hunk 3]
```diff
@@ -1,48 +1,39 @@
...
+{:.caution}
+
+To update your Kaa server from version 0.9.x to 0.10.x:
+
+1. [Download](http://www.kaaproject.org/download-kaa/) the Kaa 0.10.0 Debian package.
+
+2. Download the [upgradeKaa090to0100.sh](https://github.com/kaaproject/kaa/blob/master/server/upgrade/data-migration-0.9.0-0.10.0/src/main/resources/scripts/upgradeKaa090to0100.sh) and [data-migration.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/db-migration.one-jar.jar) to upgrade the databases.
```
Comment (Kirill380) [inline doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: Why master? Return to v0.10.0. Link become broken but it is okay, cause we haven't release 0.10 version yet and so there is no such version on GitHub repo.

Comment (Kirill380) [inline doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: Why master? Return to v0.10.0. Link become broken but it is okay, cause we haven't release 0.10 version yet and so there is no such version on GitHub repo.

Comment (Kirill380) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: And please rename db-migration.one-jar.jar to data-migration_0.9.0-0.10.0.jar. I have found out recently  that name of artifact changed.

Comment (hunch7) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: Fixed.

**Comment by Kirill380** [comment]
LGTM

**Pull request merged**